README.md

```diff
@@ -19,10 +14,50 @@ includes: The [UNT Digital Library](http://digital.library.unt.edu), [The Portal
 Requirements
 ------------
 
+* Django == 1.6
+* lxml == 3.4.4
```

Comment:
We don't need to add this to the requirements section of the README because it will be be install automatically. However we do need to list whatever libs lxml requires.